google-cloud-firestore/lib/google/cloud/firestore/v1/doc/google/firestore/v1/firestore.rb

```diff
@@ -77,16 +75,14 @@ class GetDocumentRequest; end
       #   @return [true, false]
       #     If the list should show missing documents. A missing document is a
       #     document that does not exist but has sub-documents. These documents will
-      #     be returned with a key but will not have fields,
-      #     {Google::Firestore::V1::Document#create_time Document#create_time}, or
-      #     {Google::Firestore::V1::Document#update_time Document#update_time} set.
+      #     be returned with a key but will not have fields, {Google::Firestore::V1::Document#create_time Document#create_time},
```

Comment:
Is removing the documentation line breaks really desirable?

Reply:
I think it is a by product of changes in the upstream proto or the yard documentation formatter.